lib/linguist/languages.yml

```diff
@@ -6897,12 +6897,12 @@ TI Program:
   ace_mode: text
   color: "#A0AA87"
   extensions:
+  - ".83p"
+  - ".83p.txt"
```

Comment:
🤔 Now I look at this more closely, search is returning [two](https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.83p) and [zero](https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.83p.txt) results respectively. This doesn't meet our usage requirements.

Reply:
I'll leave this PR open for the syntax highlighting and it's also still worth removing .8xk and .8xk.txt. I'll probably also go ahead and add some samples for .8xp since it looks like only .8xp.txt has samples currently.